docs/SUMMARY.md

```diff
@@ -3,63 +3,52 @@
 
 * [👋 Introduction](README.md)
 * [⬇️ Installation](https://earthly.dev/get-earthly)
-* [🔀 Alternative installation](/alt-installation.md)
 * [🎓 Learn the basics](./basics.md)
-* [⚙️ CI integration](./ci-integration.md)
-
-## 🧑‍🏫 Guides
-
-* [Authenticating Git and image registries](guides/auth.md)
-* [Target, artifact and command referencing](guides/target-ref.md)
-* [User-defined commands (UDCs)](guides/udc.md)
-* [Managing cache](guides/cache.md)
-* [Advanced local caching](guides/advanced-local-caching.md)
-* [Shared cache](guides/shared-cache.md)
-* [Using Docker in Earthly](guides/docker-in-earthly.md)
-* [Cloud secrets](guides/cloud-secrets.md)
-* [Integration Testing](guides/integration.md)
-* [Debugging techniques](guides/debugging.md)
-* [Multi-platform builds](guides/multi-platform.md)
-* Configuring registries
-    * [AWS ECR](guides/registries/aws-ecr.md)
-    * [GCP Artifact Registry](guides/registries/gcp-artifact-registry.md)
-    * [Azure ACR](guides/registries/azure-acr.md)
-    * [Self-signed certificates](guides/registries/self-signed.md)
-
-## 🌍 Earthfiles
 
+## 📖 Docs
+
+* Guides
+    * [Authenticating Git and image registries](guides/auth.md)
+    * [Target, artifact and command referencing](guides/target-ref.md)
+    * [User-defined commands (UDCs)](guides/udc.md)
+    * [Managing cache](guides/cache.md)
+    * [Advanced local caching](guides/advanced-local-caching.md)
+    * [Shared cache](guides/shared-cache.md)
+    * [Using Docker in Earthly](guides/docker-in-earthly.md)
+    * [Cloud secrets](guides/cloud-secrets.md)
+    * [Integration Testing](guides/integration.md)
+    * [Debugging techniques](guides/debugging.md)
+    * [Multi-platform builds](guides/multi-platform.md)
+    * Configuring registries
+        * [AWS ECR](guides/registries/aws-ecr.md)
+        * [GCP Artifact Registry](guides/registries/gcp-artifact-registry.md)
+        * [Azure ACR](guides/registries/azure-acr.md)
+        * [Self-signed certificates](guides/registries/self-signed.md)
 * [Earthfile reference](earthfile/earthfile.md)
```

Comment:
it looks like `earthfile/earthfile.md` was deleted in this PR, now this links to a broken page?

Reply:
Hmm no it wasn't.. here's the page in preview: https://app.gitbook.com/@earthly/s/earthly-staging/~/revisions/-MXEhZYiuUi6qVdZamDG/v/vlad%2Fdocs-toc/docs/earthfile

Reply:
well that's odd; I have no idea why I'm seeing this under the PR:

![image](https://user-images.githubusercontent.com/1806823/113363462-c5f1a680-9305-11eb-82ab-588c55ca6ef2.png)

... sigh that's github being github and not showing the change. Sorry my mind just registered it as being deleted. I'll take a look at it.

Reply:
It's almost all just reordering of the commands.